server/.env

```diff
@@ -1,4 +1,5 @@
 SUPERSEC= password123
 CLOUD_NAME=e-commerce-website
 CLOUD_API_KEY=742572971787234
-CLOUD_API_SECRET=4ytWgJj4DLzBDJNh2_J1kL2mpUU
\ No newline at end of file
+CLOUD_API_SECRET=4ytWgJj4DLzBDJNh2_J1kL2mpUU
```

Comment:
why is your .env here??

Reply:
i did not add it well in my gitignore. I have done it now sir